tests/package.json

```diff
@@ -36,22 +36,26 @@
     "watch": "npm-watch",
     "build": "cargo build --release",
     "non-ci-test": "mocha -r ts-node/register 'non_ci_tests/**/test-*.ts'",
-    "test-single": "mocha -r ts-node/register 'tests/test-parachain-bond.ts'",
+    "test-single": "mocha -r ts-node/register 'tests/test-precompile-staking.ts'",
     "current-test": "mocha -r ts-node/register",
     "lint": "npx prettier --write --ignore-path .gitignore '**/*.(yml|js|ts|json)'"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@polkadot/api": "^5.6.1",
-    "@polkadot/keyring": "^7.2.1",
-    "@polkadot/types": "^5.6.1",
-    "@polkadot/util": "^7.2.1",
-    "@polkadot/util-crypto": "^7.2.1",
+    "@polkadot/api": "^5.8.3",
+    "@polkadot/keyring": "^7.3.1",
+    "@polkadot/types": "^5.8.3",
+    "@polkadot/util": "^7.3.1",
+    "@polkadot/util-crypto": "^7.3.1",
+    "@substrate/txwrapper": "^7.0.1",
+    "@substrate/txwrapper-registry": "^1.2.13",
+    "@substrate/txwrapper-substrate": "^1.2.13",
     "@types/chai": "^4.2.11",
     "@types/mocha": "^8.2.2",
     "chai": "^4.3.4",
     "ethers": "^5.1.0",
+    "lodash": "^4.17.21",
```

Comment:
Why do we need lodash ?

Reply:
```
Error: Cannot find module 'lodash/merge'
Require stack:
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/@sora-substrate/type-definitions/build/index.js
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/@polkadot/apps-config/api/spec/soraSubstrate.cjs
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/@polkadot/apps-config/api/spec/index.cjs
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/@polkadot/apps-config/api/index.cjs
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/@substrate/txwrapper-registry/lib/index.js
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/tests/test-balance-transfer-txwrapper-substrate.ts
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/mocha/lib/esm-utils.js
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/mocha/lib/mocha.js
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/mocha/lib/cli/one-and-dones.js
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/mocha/lib/cli/options.js
- /Users/antoineestienne/GitHubRepo/moonbeam/tests/node_modules/mocha/bin/mocha
```
It looks like it's part of the dependencies of @polkadot/apps-config, which is itself needed to get the registry

Reply:
Why is it added in our package.json ? If it is a depepndency of a library it should be in that library package.json

Reply:
You're right. Created an issue: https://github.com/paritytech/txwrapper-core/issues/130